docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs

```diff
@@ -80,23 +80,27 @@ public static void PFI_Regression()
 
             // Get the feature indices sorted by their impact on R-Squared
             var sortedIndices = permutationMetrics.Select((metrics, index) => new { index, metrics.RSquared })
-                .OrderByDescending(feature => Math.Abs(feature.RSquared))
+                .OrderByDescending(feature => Math.Abs(feature.RSquared.Mean))
```

Comment:
>Mean [](start = 72, length = 4)

potentially add a clause that uses stdiv  in meaningful way?

Reply:
I print out the standard error of the mean in the example, to show that the differences in the r-squared detected in PFI are meaningful.